app/models/product.rb

```diff
@@ -10,7 +10,7 @@ class Product < ActiveRecord::Base
   has_many   :stored_files
   has_many   :price_groups, through: :price_group_products
   has_many   :price_group_products
-  has_many :product_accessories, -> { where(deleted_at: nil) }, dependent: :destroy
+  has_many   :product_accessories, -> { where(deleted_at: nil) }, dependent: :destroy
```

Comment:
If this is just a spacing change, I would not make it, as ideally, the not aligning becomes the default.